.github/workflows/nightly-test-integrations-1.15.x.yml

```diff
@@ -50,6 +50,14 @@ jobs:
   get-go-version:
```

Comment:
The next few files are identical other than the sourced `release/...` branch for getting the Envoy version, and which Consul versions they test against.